examples/oneapi/dpc/source/decision_forest/df_reg_traverse_model.cpp

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#define ONEDAL_DATA_PARALLEL
+#include "oneapi/dal/algo/decision_forest.hpp"
+#include "oneapi/dal/io/csv.hpp"
+
+#include "example_util/utils.hpp"
+#include "oneapi/dal/exceptions.hpp"
+
+namespace dal = oneapi::dal;
+namespace df = dal::decision_forest;
+
+/** Visitor class, prints out tree nodes of the model when it is called back by model traversal method */
+template <typename Task>
+struct print_node_visitor {
+    bool operator()(const df::leaf_node_info<Task> &info) {
+        for (std::int64_t i = 0; i < info.get_level(); ++i)
+            std::cout << "  ";
```

Comment:
```suggestion
        std::cout << std::string{ info.get_level(), "  " };
```

https://stackoverflow.com/questions/7897163/stdcout-to-print-character-n-times

Reply:
fixed